src/core_plugins/metric_vis/public/components/metric_vis_value.js

```diff
@@ -26,6 +26,14 @@ class MetricVisValue extends Component {
       'metric-container--filterable': hasFilter
     });
 
+    /*
+     * Justification for dangerouslySetInnerHTML:
```

Comment:
How would you feel about putting this comment between the `style` and `dangerouslySetInnerHTML` attributes so it is really annoying?

Reply:
I can do annoying 👍